backend/graphql/resolvers/authResolvers.ts

```diff
@@ -31,16 +31,6 @@ const authResolvers = {
       res.cookie("refreshToken", refreshToken, cookieOptions);
       return rest;
     },
-    loginWithGoogle: async (
```

Comment:
Removed since we won't use Google OAuth and having it messes up some of the changes in this ticket.